src/components/BrancheView.vue

```diff
@@ -646,7 +660,27 @@ export default {
 
       });
     },
-
+    async downloadImage(fileId){
+      let result = await this.downloadImageFile(fileId);
+      let blob = new Blob([result.data], {type: 'image/*'});
+      let url = URL.createObjectURL(blob);
+      this.$refs.branchImage.src = url;
+    },
+    async downloadDocument(file){
```

Comment:
please use httpClient instead of using Axios directly

Reply:
done